src/components/organisms/curriculum/OakDownloadsJourneyChildSubjectTierSelector/OakDownloadsJourneyChildSubjectTierSelector.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from "react";
+import styled, { css } from "styled-components";
+
+import { OakBox, OakHeading } from "../../../atoms";
+
+// import { parseSpacing } from "@/styles/helpers/parseSpacing";
+import {
+  OakPrimaryButton,
+  OakRadioButton,
+  OakRadioGroup,
+} from "@/components/molecules";
+
+// const StyledOakFlex = styled(OakFlex)`
+//   width: ${parseSpacing("all-spacing-6")};
+// `;
+// For example you could restyle the OakFlex component by adding the styles to the css template literal below
+export interface Tier {
+  tier: string;
+  tier_slug: string;
```

Comment:
it's interesting that you guys are working with snake_case here. My view is that it should be snake_case at DB level and then we should convert immediatetly and consistently to camelCase for React world. We've been doing this already on our queries in OWA>

Reply:
Good point, it makes sense to update this to camel case for front end, I just hadn't noticed when copying the types across.. Will update in the component and make some tickets in OWA for consistency